src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1048,7 +1048,14 @@ private function addCacheSection(ArrayNodeDefinition $rootNode)
                                     ->end()
                                     ->scalarNode('tags')->defaultNull()->end()
                                     ->booleanNode('public')->defaultFalse()->end()
-                                    ->integerNode('default_lifetime')->end()
+                                    ->integerNode('default_lifetime')
```

Comment:
This must be a `scalarNode`, right?

Reply:
`beforeNormalization` will happen before the type is applied, so no